Cargo.toml

```
@@ -102,3 +114,7 @@ required-features = ["rt", "stm32f303xc", "stm32-usbd"]
 [[example]]
 name = "spi"
 required-features = ["stm32f303"]
+
+[[example]]
+name = "can"
+required-features = ["rt", "can", "device-selected"]
```

As the [example](https://github.com/stm32-rs/stm32f3xx-hal/pull/100/files#diff-f005bfff7df113692409278f2a71eec9R7) explicitly uses the `stm32f303` we should requre the "stm32f303" feature instead of the "device-sected" feature: 

```suggestion
required-features = ["rt", "can", "stm32f303"]
```

"device-seletected" feature is not really meant to be used outside the crate and is an implementation detail.